### PR TITLE
Update kubeadm-init.md

### PR DESCRIPTION
**Pull request by jihoon-seo**
According to [API v1beta1](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1), 
"unifiedControlPlaneImage" in ClusterConfiguration was changed to "useHyperKubeImage".

```
A list of changes since v1alpha3:
...
- "unifiedControlPlaneImage" in ClusterConfiguration was changed to a boolean field called "useHyperKubeImage".
...
```

This PR removes description for the old `unifiedControlPlaneImage`  in using custom images by kubeadm configuration file.
and adds description for "useHyperKubeImage".

### CODE REVIEW

**Comment by jihoon-seo** [comment]
This is the preview for the changed section: 
https://deploy-preview-13068--kubernetes-io-master-staging.netlify.com/docs/reference/setup-tools/kubeadm/kubeadm-init/#custom-images

**Comment by jihoon-seo** [comment]
This is the preview for the second commit: 
https://deploy-preview-13068--kubernetes-io-master-staging.netlify.com/docs/reference/setup-tools/kubeadm/kubeadm-init/#running-kubeadm-without-an-internet-connection

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ the [kubeadm config migrate](/docs/reference/setup-tools/kubeadm/kubeadm-config/
 because `v1alpha3` will be removed in Kubernetes 1.14.
 
 For more details on each field in the `v1beta1` configuration you can navigate to our
-[API reference pages.] (https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1)
+[API reference pages](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1).
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: this seems like a good change.

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 2]
```diff
@@ -148,8 +148,23 @@ Allowed customization are:
 
 * To provide an alternative `imageRepository` to be used instead of
   `k8s.gcr.io`.
-* To provide a `unifiedControlPlaneImage` to be used instead of different images for control plane components.
-* To provide a specific `etcd.image` to be used instead of the image available at`k8s.gcr.io`.
+* To set `useHyperKubeImage` to `true` to use images from HyperKube.
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: please change to:
> To set `useHyperKubeImage` to `true` to use the HyperKube image.

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 3]
```diff
@@ -148,8 +148,23 @@ Allowed customization are:
...
+* (For `etcd` only) To provide a specific `imageRepository` and `imageTag` to be used instead of the image available at`k8s.gcr.io`.
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: please change to:
> To provide a specific `imageRepository` and `imageTag` for etcd

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 4]
```diff
@@ -148,8 +148,23 @@ Allowed customization are:
...
+
+An example from [kubeadm API v1beta1](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1#hdr-Kubeadm_init_configuration_types):
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: please remove this example. this link:
https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1

already has examples.

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 5]
```diff
@@ -347,20 +362,19 @@ In summary, `kubeadm alpha selfhosting` works as follows:
...
+| k8s.gcr.io/kube-controller-manager         | v1.13.x                      |
+| k8s.gcr.io/kube-scheduler                  | v1.13.x                      |
+| k8s.gcr.io/kube-proxy                      | v1.13.x                      |
+| k8s.gcr.io/pause                           | 3.1                          |
+| k8s.gcr.io/etcd                            | 3.2.24                       |
+| k8s.gcr.io/coredns                         | 1.2.6                        |
+
+Here `v1.13.x` means the "latest patch release of the v1.13 branch".
+
+For Kubernetes version 1.10 or earlier, `-${ARCH}` suffix is required for all image names except `coredns/coredns`.
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: this table was provided as a workaround for 1.10 and now is a good time to remove it.
but we have to do it in a separate PR against the dev-1.14 branch.

please remove the change from this PR.

**Comment by jihoon-seo** [comment]
Thank you @neolit123 !
And this is the preview for the fourth commit:
https://deploy-preview-13068--kubernetes-io-master-staging.netlify.com/docs/reference/setup-tools/kubeadm/kubeadm-init/#custom-images

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 6]
```diff
@@ -148,8 +148,8 @@ Allowed customization are:
 
 * To provide an alternative `imageRepository` to be used instead of
   `k8s.gcr.io`.
-* To provide a `unifiedControlPlaneImage` to be used instead of different images for control plane components.
-* To provide a specific `etcd.image` to be used instead of the image available at`k8s.gcr.io`.
+* To set `useHyperKubeImage` to `true` to use the HyperKube image.
+* To provide a specific `imageRepository` and `imageTag` for etcd.
```
Comment (rosti) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: `for etcd or DNS add-on.`

**Comment by cody-clark** [comment]
Thanks so much, @jihoon-seo!

/approve
